src/Operation/Find.php

```diff
@@ -292,12 +292,20 @@ public function __construct(string $databaseName, string $collectionName, $filte
             unset($options['readConcern']);
         }
 
-        if (isset($options['snapshot'])) {
-            trigger_error('The "snapshot" option is deprecated and will be removed in a future release', E_USER_DEPRECATED);
+        if (isset($options['maxScan'])) {
+            @trigger_error('The "maxScan" option is deprecated and will be removed in a future release', E_USER_DEPRECATED);
         }
 
-        if (isset($options['maxScan'])) {
-            trigger_error('The "maxScan" option is deprecated and will be removed in a future release', E_USER_DEPRECATED);
+        if (isset($options['modifiers'])) {
+            @trigger_error('The "modifiers" option is deprecated and will be removed in a future release', E_USER_DEPRECATED);
+        }
+
+        if (isset($options['oplogReplay'])) {
+            @trigger_error('The "oplogReplay" option is deprecated and will be removed in a future release', E_USER_DEPRECATED);
+        }
```

Comment:
Excluding `modifiers`, PHPC raises its own `E_DEPRECATED` notice these options in the Query constructor. Does that mean users would see two deprecation notices?

As for `modifiers`, that seems like something we should add in PHPC 1.x before we remove the option entirely in 2.0 ([PHPC-2440](https://jira.mongodb.org/browse/PHPC-2440)). I created [PHPC-2452](https://jira.mongodb.org/browse/PHPC-2452) and opened https://github.com/mongodb/mongo-php-driver/pull/1677 to address that.

Reply:
Yes, that would cause duplicate deprecation notices, so we probably should remove the extra notices thrown in PHPLIB. I'll comment on https://github.com/mongodb/mongo-php-driver/pull/1677 regarding the `modifier` deprecations, but I'll remove the query option deprecations entirely in this PR.

Reply:
I repurposed this PR to remove the existing deprecation notices for query options instead of adding new ones, as PHPC already emits deprecation notices for deprecated options. The mapReduce deprecation was left in place.